src/server.c

```diff
@@ -3530,8 +3591,10 @@ void createSharedObjects(void) {
         "-NOSCRIPT No matching script. Please use EVAL.\r\n"));
     shared.loadingerr = createObject(OBJ_STRING,sdsnew(
         "-LOADING Redis is loading the dataset in memory\r\n"));
-    shared.slowscripterr = createObject(OBJ_STRING,sdsnew(
+    shared.slowevalerr = createObject(OBJ_STRING,sdsnew(
         "-BUSY Redis is busy running a script. You can only call SCRIPT KILL or SHUTDOWN NOSAVE.\r\n"));
+    shared.slowscripterr = createObject(OBJ_STRING,sdsnew(
+        "-BUSY Redis is busy running a script. You can only call FUNCTION KILL or SHUTDOWN NOSAVE.\r\n"));
```

Comment:
We could also tell the user what script is being run because these have names, that seems useful?

Reply:
Currently the run_ctx is shared between eval and functions and it is not aware about the functions which is currently running (his purpose is to interact with Redis from within a script). I can add optional metadata that can be added there and we can put the running function in this metadata. Then we will be able to reply with the function name, WDYT?

Reply:
we only recently added something like that for EVAL, we print the name to the log.
either way, i don't think it should be part of the error message.
maybe a different introspection command that is allowed to run during a busy script?

Reply:
I'm fine either way, I just think the user should have a way to know what "Function" is running.

Reply:
We discussed it on the issue (https://github.com/redis/redis/issues/8693#issuecomment-860160430), @oranagra mentioned that its not enough to know the function that is running because it might be change by the time we will send the `FUNCTION KILL`. Even if we will allow to kill only if the function name is equal to a given parameter, it still might have finished and restarted by the time we will send the command.
Regardless, we can add `FUNCTION RUNNING` that will return the current running function (as mentioned on the PR message), I am just not sure it will help the user kill the "right" function. Let me know what you think.

Reply:
I agree, it should be some sort of introspection command, (not part of the error)
maybe instead of FUNCTION RUNNING, we can add some FUNCTION STATUS or STATS that will print some general info on all engines (unlike FUNCTION INFO which is on one function, right?).
i.e. something similar to MEMORY STATS and INFO.
there was can put all soft of general details, including the currently running function, it's args, and execution duration.

Reply:
Added FUNCTION STATS command, update the top comment with the relevant info.